### PR TITLE
Add display names for ci steps

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ pool:
   vmImage: 'macos-latest'
 steps:
 - task: DownloadGitHubRelease@0
+  displayName: 'Download MacOS Server'
```
Comment (anthonylavado) [inline azure-pipelines.yml]:
```suggestion
  displayName: 'Download macOS Server'
```

File: azure-pipelines.yml [hunk 2]
```diff
@@ -10,40 +11,47 @@ steps:
     downloadPath: '$(System.WorkFolder)'
 
 - task: ExtractFiles@1
+  displayName: 'Extract MacOS Server'
```
Comment (anthonylavado) [inline azure-pipelines.yml]:
```suggestion
  displayName: 'Extract macOS Server'
```

File: azure-pipelines.yml [hunk 3]
```diff
@@ -10,40 +11,47 @@ steps:
...
 
 - task: Bash@3
+  displayName: 'Copy FFprobe'
   inputs:
     targetType: 'inline'
     script: 'mv $(Build.SourcesDirectory)/ffmpegtemp/ffmpeg*/bin/ffprobe $(Pipeline.Workspace)/jellyfin-mac-app-resources/'
     failOnStderr: true
 
 - task: Xcode@5
+  displayName: 'Build MacOS Wrapper'
```
Comment (anthonylavado) [inline azure-pipelines.yml]:
```suggestion
  displayName: 'Build macOS Wrapper'
```

**Review by anthonylavado — CHANGES_REQUESTED** [review]
Sorry 😐

File: azure-pipelines.yml (hunk 1 above)
Comment (dkanada) [reply azure-pipelines.yml]: doesn't title casing supercede product names? honestly have no idea haha

Comment (dkanada) [reply azure-pipelines.yml]: the pipeline name on azure is `Mac` so we should change one of them at least

Comment (thornbill) [reply azure-pipelines.yml]: I think product names always take precedence.

Comment (dkanada) [reply azure-pipelines.yml]: Alright, whoever has access to Azure should change the pipeline to `Jellyfin Server macOS` to match this then.

Comment (anthonylavado) [reply azure-pipelines.yml]: @dkanada Will do.

**Review by anthonylavado — APPROVED** [review]

**Pull request merged**